### DISCUSSION TITLE
Ploting && farming on 2 PCs, with same mnemonic

### DISCUSSION THREAD

**Discussion by ch3ll0v3k** [Chia-Network/chia-blockchain#2507] [General]
Hi, i have 2 PC, and account with the same  mnemonic.

I have plotted the plot, and after transferred to other PC, but it says "Plots not found, check your connection".

Is the final plot somehow connected to hardware ?

**Comment by dorkmo** [comment]
Need to add the plot directory on the new machine

**Reply by ch3ll0v3k** [reply]
yes, i did. nothing helps.

only full restart of software has resolved the issue

**Reply by casa10** [reply]
Hello,
I am interested about how you did to have 2 PC with same account.
Do you :
- install on the first chia GUI and created account, take your 24 words and keys on one file
- install chia gui on the second computer and import the mnemonic
- open chia gui on the second computer and go to "Plots" -> add plot directory and add the hards drives to your farm 
and after that worked ?
You can farm with the two computers so ?
Thank you for your help

**Reply by kfarzad** [reply]
I have the very same question as casa10. Also, my network is the same for both PCs. Please help me.

**Reply by DylannJ** [reply]
> Hello,
> I am interested about how you did to have 2 PC with same account.
> Do you :
> 
> * install on the first chia GUI and created account, take your 24 words and keys on one file
> * install chia gui on the second computer and import the mnemonic
> * open chia gui on the second computer and go to "Plots" -> add plot directory and add the hards drives to your farm
>   and after that worked ?
>   You can farm with the two computers so ?
>   Thank you for your help

I don't know for ch3ll0v3k but I do exatly what you saying, but the thing is they not have same Farm Overview, my laptop say 5years to win and my desktop 2years, How can I combine this 2 pc connected to the same account for reducing the time to win ?

**Reply by kfarzad** [reply]
Could you check this video? The owner says we should disable upnp on the second machine. My second PC has not arrived yet, so I cannot check it myself rightnow. I will check in a couple of days.
https://www.youtube.com/watch?v=wEqk2AbpdNg&lc=UgxqRevGuqxI_4ex2fJ4AaABAg.9MfMbf94YmX9MfUPJ56fnk

**Reply by StogieC** [reply]
https://github.com/Chia-Network/chia-blockchain/discussions/1116#discussioncomment-420398

**Reply by khvahik** [reply]
> Could you check this video? The owner says we should disable upnp on the second machine. My second PC has not arrived yet, so I cannot check it myself rightnow. I will check in a couple of days.
> https://www.youtube.com/watch?v=wEqk2AbpdNg&lc=UgxqRevGuqxI_4ex2fJ4AaABAg.9MfMbf94YmX9MfUPJ56fnk

Kiarash, did you checked it?

**Reply by markecx** [reply]
I have tried plotting different plots on 3 PC's at the same time in hope that it will be faster but it's not :(

I just entered the mnemonic phase and it recognised my private key. Once the plot was finished, I copied into the main PC directory, refreshed the plots and the plot was visible in the list.

**Reply by kyawpaingthet** [reply]
I think upnp disable is more effective way to setup 2pc for ploting and farming.Why upnp is disable because if both PC are farming on same memonic seed your chia wallet  confusing about calculate the Estimate winning time,so disable farming on ploting PC and only farming on farming PC.Just my Thought.if any different thought please let me know.
